src/platforms/javascript/common/session-replay/troubleshooting.mdx

```diff
@@ -62,6 +62,20 @@ Because of the complexity of the browser environment, there's a significant amou
 
 We're always working on ways to reduce the bundle size. Additionally, there are steps you can take in order to reduce the size of Session Replay based on your specific use case. See [Tree Shaking](../../configuration/tree-shaking/) for more information.
 
+## How does Rage Click detection work?
+
+The Replay SDK tries to detect cases where a user clicked on something, but nothing happened. This is called a "Dead Click", a click that seems to be "dead" and does nothing, from the users perspective. This can happen if a bug or similar prevents the UI from being correctly updated after a button is clicked.
+
+The way the Replay SDK detects this is by checking if a click on a `<button>` or `<a>` element does not lead to any updates of the DOM, or a page scroll. If within 7 seconds no updates are detected, the SDK will generate a "Slow Click" in the Replay UI. We call it a "Slow Click" because it may be possible that something will happen after the 7s, but we chose this as a cutoff time.
+
+Another concept related to this is "Rage Clicks". A "Rage Click" is when a user clicks many times on an element, which may indicate frustration that something is not working.
+
+When a Slow Click and a Rage Click come together, we have a very good indicator that something is actually wrong, and the Replay SDK will generate a "Rage Click" for this.
+
+Note that there are some limitations to this. On the one hand, there may be _false negatives_ - we cannot detect all cases, for example, imagine a user clicks a "dead" button, but then clicks a different button afterwards that leads to a DOM update. In this case, since we cannot attribute the DOM update to a specific button, we'll _not_ capture this slow click.
```

Comment:
```suggestion
Note that there are some limitations to this. On the one hand, there may be _false negatives_ -- we cannot detect all cases; for example, imagine a user clicks a "dead" button, but then clicks a different button afterwards that leads to a DOM update. In this case, since we cannot attribute the DOM update to a specific button, we'll _not_ capture this slow click.
```